src/quo2/components/settings/settings_list/view.cljs

```diff
@@ -0,0 +1,119 @@
+(ns quo2.components.settings.settings-list.view
+  (:require [quo2.components.settings.settings-list.style :as style]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]))
+
+(defn settings-title
+  [title]
+  [rn/view
+   {:flex 1}
+   [text/text
+    {:accessibility-label :setting-item-name-text
+     :ellipsize-mode      :tail
+     :style               style/title
+     :number-of-lines     1
+     :weight              :medium
+     :size                :paragraph-1}
+    title]
+  ])
+
+(def browser-context-icon
+  [rn/view
+   [quo2.icons/icon :browser-context
+    {:container-style style/icon
+     :color           (colors/theme-colors
+                       colors/neutral-50
+                       colors/neutral-40)}]])
+
+(def chevron-icon
+  [rn/view
+   [quo2.icons/icon :chevron-right
+    {:color (colors/theme-colors
+             colors/neutral-50
+             colors/neutral-40)}]])
+
+(defn switch-button
+  [{:keys [:value
+           on-value-change]}]
+  (js/console.warn on-value-change)
+  [rn/switch
+   {:value                value
+    :on-value-change      (fn [new-value] (on-value-change new-value))
+    ;; TODO: For some reason this prop is not working on ios simulator, spike a little bit more
+    :ios-background-color (colors/theme-colors
+                           colors/neutral-30
+                           colors/neutral-80)
+    ;; TODO: Ask for colors on the track, there is no definition on the design
+    ;; Colors for false value only work on Android
+    ;; :track-color          {:false (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)
+    ;;                       :true  (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)}
+    :style                style/switch-button}]
+)
+
+(def badge-icon
+  [rn/view
+   {:style style/dot}])
+
+(defn right-button
+  [{:keys [title
+           on-press]}]
+  [rn/touchable-opacity
+   {:on-press on-press
```

Comment:
better to use `quo2/button` if possible

Reply:
@rfahernandez In the design we are using [this variant quo2 button](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=9-41&t=D5N6jmSMTa3ARAV4-4). 

In Figma, you can right-click on the component and choose the `Got to main component` option to see the version of the existing components we are using.
<img width="1728" alt="Screenshot 2023-05-17 at 10 20 29" src="https://github.com/status-im/status-mobile/assets/19279756/fb8d1e25-2345-490f-aee8-8a4bb2c27ceb">

Same thing you can do for the Switch, Communities, Icons, etc. And instead of recreating those components, you can use them directly from quo2